lib/compile/apply-plugins.js

```diff
@@ -0,0 +1,17 @@
+module.exports = function applyPlugins (plugins, ev /* args... */) {
+  if (!ev) {
```

Comment:
I haven't gotten far enough to see if you use Event Emitters elsewhere but at a glance the name `ev` and `event` might not make sense here since it's actually a string representing a lifecycle phase and not an event of any kind. For what it's worth I had to go figure out where the event was coming from while scratching my head :)